controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -662,10 +656,10 @@ func (r *DataScienceClusterReconciler) getRequestName(ctx context.Context) (stri
 // argoWorkflowCRDPredicates filters the delete events to trigger reconcile when Argo Workflow CRD is deleted.
 var argoWorkflowCRDPredicates = predicate.Funcs{
 	DeleteFunc: func(e event.DeleteEvent) bool {
-		if e.Object.GetName() == datasciencepipelines.ArgoWorkflowCRD {
+		if e.Object.GetName() == datasciencepipelinesctrl.ArgoWorkflowCRD {
```

Comment:
i am thinking, should not this `argoWorkflowCRDPredicates` moved into component, instead of having in DSC controller?

Reply:
Yeah this can probably be moved to the component will do now

Reply:
we can keep it for now, i think some improvement ticket can follow cleanup case later